contracts/root/RootChainManager/RootChainManager.sol

```diff
@@ -187,7 +187,7 @@ contract RootChainManager is IRootChainManager, Initializable, AccessControl {
             abi.encodePacked(
                 inputDataRLPList[2].toBytes(), // blockNumber
                 inputDataRLPList[6].toBytes(), // receipt
-                inputDataRLPList[9].toBytes() // logIndex
+                inputDataRLPList[9].toUint() // logIndex
```

Comment:
Use block number (as `uint`, not `bytes`) and log index as a unique Id. No need to add receipt bytes.

```
inputDataRLPList[2].toUint(), // block number
inputDataRLPList[9].toUint() // log index
```

Reply:
@arthcp said log index is actually `logIndexInReceipt` not `logIndexInBlock`. So we need transaction index or unique receipt (receipt bytes).

Reply:
@jdkanani so i'll just change the block number to uint for now?
cc: @arthcp

Reply:
@AyushKaul change block number to uint and use branch mask instead of receipt.